alert-common/src/main/java/com/synopsys/integration/alert/common/rest/model/FieldModel.java

```diff
@@ -78,6 +78,10 @@ public void putField(final String key, final FieldValueModel field) {
         keyToValues.put(key, field);
     }
 
+    public void removeField(final String key) {
+        keyToValues.remove(key);
+    }
```

Comment:
Since events now have the opportunity modify the FieldModel content, we can no longer pass unmodifiable items to keyToValues. I was not able to find a better alternative for using events to modify FieldModels.